### PR TITLE
bump gcc dependencies

### PR DESCRIPTION
**Pull request by AndreyMlashkin**
Specify library name and version:  **gcc/10.2**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by AndreyMlashkin** [comment]
With my fix for system libs, it's not a version bump anymore, so I need an approval
@prince-chrismc @SSE4 @uilianries @dmn-star 
please approve

**Comment by uilianries** [comment]
@AndreyMlashkin Thank you for your contribution. Please, take a look on the linter errors. That recipe is outdated, based on Conan 2.0

**Comment by uilianries** [comment]
@AndreyMlashkin Please, consider https://github.com/AndreyMlashkin/conan-center-index/pull/8

File: recipes/gcc/all/conanfile.py [hunk 1]
```diff
@@ -108,9 +110,10 @@ def package(self):
             autotools.install(args=self._make_args)
         else:
             autotools.make(args=["install-strip"] + self._make_args)
-        tools.rmdir(os.path.join(self.package_folder, "share"))
-        tools.remove_files_by_mask(self.package_folder, "*.la")
-        self.copy(pattern="COPYING*", dst="licenses", src=self._source_subfolder)
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.la", self.package_folder)
```
Comment (uilianries) [inline recipes/gcc/all/conanfile.py:114]:
```suggestion
        rm(self, "*.la", self.package_folder, recursive=True)
```
Need to go deeper to find .la files

File: recipes/gcc/all/test_package/conanfile.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-from conans import ConanFile, tools
+from conan import ConanFile, tools
```
Comment (uilianries) [inline recipes/gcc/all/test_package/conanfile.py:1]:
```suggestion
from conan import ConanFile
from conans import tools
```

**Pull request closed without merging**